web/themes/custom/hatter/hatter.libraries.yml

```diff
@@ -4,8 +4,15 @@ global-css:
       css/print.css:
         media: print
       css/styles.css: {}
-global-js:
-  dependencies:
-    - core/jquery
```

Comment:
Do we not need this any more? I don't see it added back.

Reply:
We don't have any global js scripts